src/Wrapper.js

```diff
@@ -39,21 +39,24 @@ class Wrapper extends React.Component {
       this.handleTransaction(transactionBag)
     }
   }
-  openApp = (appId, params) => {
+  openApp = (proxyAddress, params) => {
```

Comment:
We already have activeInstanceId. I think it is ok to use proxyAddress for labeling static apps, and just let apps like home, settings, and permissions use that as their proxyAddress. The current way leaves us comparing activeInstanceId to proxyAddress in two places. Trying to use instanceId would result in us comparing instanceId to proxyAddress every single time we try to find an app by its instanceId.

Reply:
> We already have `activeInstanceId`.

`activeInstanceId` was precisely refering to `instanceId` 😁

A solution would be to change these to `activeProxyAddress`, but having things  named `proxyAddress`, that are not proxy addresses, sounds too confusing IMO.

> Trying to use instanceId would result in us comparing instanceId to proxyAddress every single time we try to find an app by its instanceId.

It’s only about naming. To get an app from a proxy address, instead of doing:

`apps.find(app => app.proxyAddress === proxyAddress)`

We’ll do:

`apps.find(app => app.instanceId === proxyAddress)`